Utils/github_workflow_scripts/delete_stale_non_contrib_branches.py

```diff
@@ -40,12 +40,8 @@ def get_non_contributor_stale_branch_names(repo: Repository) -> List[str]:  # no
             if (last_modified := branch.commit.commit.last_modified) and (
                     last_commit_datetime := parse(last_modified)):
                 elapsed_days = (now - last_commit_datetime).days
-                # print(f'{elapsed_days=}')
                 if elapsed_days >= 60:
-                    associated_open_prs = branch.commit.get_pulls()  # type: ignore[attr-defined]
-                    associated_open_prs = [pr for pr in associated_open_prs if pr.state == 'open']
-                    if len(associated_open_prs) < 1:
-                        branch_names.append(branch.name)
+                    branch_names.append(branch.name)
```

Comment:
merge the if statement by also assigning the value to  elapsed_days using walrus operator and comparing it to 60.